cmd/minishift/cmd/profile/profile_set.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 const (
-	unableToSetOcContextErrorMessage = "Make sure the profile is in running state or restart if the problem persists."
+	vmDoesNotExist = "Does Not Exist"
```

Comment:
I did not have a local constant in mind, but rather a shared/global constant somewhere in our cmd utility package. This way we can use this constant anywhere we need to compare against the state. 

In fact when it comes to actual libmachine states there are constants for them in github.com/docker/machine/libmachine/state. It is just the "Does not exist" which is the issue. In the cmd util package we have helpers for IsHostRunning. I would expect something like DoesHostExist.